e3sm_warehouse/e3sm_warehouse.py

```diff
@@ -16,13 +17,13 @@
 from termcolor import colored, cprint
 
 
-from warehouse.workflows import Workflow
-from warehouse.dataset import Dataset, DatasetStatus
-from warehouse.slurm import Slurm
-from warehouse.listener import Listener
-import warehouse.resources as resources
-import warehouse.util as util
-from warehouse.util import setup_logging, log_message
```

Comment:
Example module import updates

Reply:
This is another example of term-overloading.  We still refer to the user_pub filesystem location, to which archives datasets are extracted, as the "warehouse" (fully, /p/user_pub/e3sm/warehouse), and the "-w" of the command line allows a user to override that location.

Reply:
This is another matter. Granted, we have no other "warehouse" project, but changing to "e3sm_warehouse" is a possibility.  Major hassle, but if it makes something cleaner, it is do-able.